helm/kubernetes-node-exporter-chart/templates/daemonset.yaml

```diff
@@ -24,6 +24,8 @@ spec:
       - key: node-role.kubernetes.io/master
         operator: Exists
         effect: NoSchedule
+      securityContext:
+        runAsUser: 0
```

Comment:
This version of node-exporter uses the `nobody` user in the Dockerfile, which doesn't have permission to read kubelet mounts (which we need to see disk space for persistent volumes) - therefore, we run as root